pkg/minishift/network/settings.go

```diff
@@ -43,31 +57,254 @@ type NetworkSettings struct {
 	Gateway   string
 	DNS1      string
 	DNS2      string
+	UseDHCP   bool
+	Disabled  bool
+}
+
+// checkSupportForAddressAssignment returns true when the instance can support
+// minishift-set-ipaddress
+func checkSupportForAddressAssignment() bool {
+	if minishiftConfig.InstanceConfig.IsRHELBased &&
+		minishiftConfig.InstanceConfig.SupportsNetworkAssignment {
+		return true
+	} else {
+		atexit.ExitWithMessage(1, configureNetworkNotSupportedMessage)
+	}
+	return false
+}
+
+// ConfigureDynamicAssignment will write configuration files which will force DHCP
+// assignment, which will be used by minishift-set-ipaddress on start of the instance.
+func ConfigureDynamicAssignment(driver drivers.Driver) {
+	if checkSupportForAddressAssignment() {
+		fmt.Println("Writing configuration for dynamic assignment of IP address")
+	}
+
+	networkSettingsEth0 := NetworkSettings{
+		Device:  "eth0",
+		UseDHCP: true,
+	}
+	WriteNetworkSettingsToInstance(driver, networkSettingsEth0)
+
+	networkSettingsEth1 := NetworkSettings{
+		Device:  "eth1",
+		UseDHCP: true,
+	}
+	WriteNetworkSettingsToInstance(driver, networkSettingsEth1)
+
+	fmt.Println(configureRestartNeededMessage)
+}
+
+// ConfigureStaticAssignment will collect NetworkSettings from the current running
+// instance and write this the values as configuration files, which will be used
+// by minishift-set-ipaddress on start of the instance.
+func ConfigureStaticAssignment(driver drivers.Driver) {
+	// Not supported for KVM
+	if minishiftConfig.IsKVM() {
+		atexit.ExitWithMessage(1, configureIPAddressFailure)
+		// related to issues with the driver (ip is retrieved from the lease)
```

Comment:
Not sure what this comment about.

Reply:
Read the documentation: KVM is not supported.

If IP is retrieved from a DHCP lease, what does that mean for a staticly assigned IP?